source/_lovelace/button.markdown

```diff
@@ -7,10 +7,35 @@ description: "The Button card allows you to add buttons to perform tasks."
 The Button card allows you to add buttons to perform tasks.
 
 <p class='img'>
-<img src='/images/lovelace/lovelace_entity_button_card.png' alt='Screenshot of the button card'>
-Screenshot of the Button card.
+<img src='/images/lovelace/lovelace_entity_button_card.png' alt='Screenshot of the button cards'>
+Screenshot of three Button Cards.
 </p>
 
+## Card Settings
+
+{% configuration_basic %}
+Entity:
+  description: The entity the card interacts with, for example `light.living_room`.
+Name:
+  description: The button name. The default is the name of the entity, if an entity is selected.
```

Comment:
```suggestion
  description: The name that will appear on the button. If not set, the name will be the name of the entity, if an entity is selected.
```

Lets try to keep the wording similar for all cards.

Reply:
I’m using the [Google style guide](https://developers.google.com/style/tense) to inform best practice here. My preference would be “The name that appears on the button”, if anything - but that seems very wordy for something that only needs a short description. Since we are updating all the card documentation (or at least I thought we were??), this might be something to change across them all.

Reply:
The setting is called Name... The button Name gives us no more detail about that settings. Something about it appearing on the button/card is good. I know its basically common sense but a name could be above the card or below etc. And otherwise we aren't giving any more details. Style it the way you want. My suggestions are more of a push to change it no exactly what it should be or what is best

Reply:
Hmmm... how about “The button name, which is displayed on the card below the icon. If this field is left blank and the card interacts with an entity, the button name defaults to the entity ID. If this field is left blank and the card does not interact with an entity, no name is displayed on the card.”

Reply:
Yea except it defaults to the name of the entity not the entity ID